src/clj_infrastructure/db.clj

```diff
@@ -661,3 +675,12 @@
         col-value-pairs (partition 2 (rest key-string-parts))
         where-conditions (map (fn [[col value]] (str col "='" value "'")) col-value-pairs)]
     (str/join " and " where-conditions)))
+
+; Setup defaults overrides in a way that prevents circular dependencies
+; (defaults that depend on not-yet defined DB lib functions)
+
+(dbconfig-override
```

Comment:
I tried every permutation of the function ordering and can't get out of the circular dependency any other way.  I'm open to a concrete alternative, but I couldn't find a simple one.